src/components/ShareTooltip/index.ts

```diff
@@ -1,4 +1,4 @@
-export {ShareSocialNetwork} from './constants';
+export {ShareOptions} from './constants';
```

Comment:
It looks like breaking change. To avoid it, it's better to reexport it with previous name too.
```ts
export {ShareOptions, ShareOptions as ShareSocialNetwork} from './constants';
```

Reply:
ok